Atomic/syscontainers.py

```diff
@@ -1628,7 +1628,8 @@ def _inspect_system_branch(self, repo, imagebranch):
         image_id = commit_rev
         id_ = None
 
-        if len(branch_id) == 64:
+        is_layer_ref = SystemContainers.is_hex(branch_id) and len(branch_id) == 64
```

Comment:
if we don't use the `len(branch_id) == 64` condition, should we drop it here as well?

Reply:
Hmm, /me thinks. yea you are right! Thanks for pointing it out. Before I was worried that the decoded ref will be the case that you mentioned earlier (i.e: cafe). But thinking more about it, I realized that, when you decode from the ostree ref of a valid image, you will still get the tag, which means the branch id will contain a ":".

As a result, the valid image( image that are not layers and with valid repos / tags) will not be considered as hex due to an extra ":". 

I will have a fix for it, thanks.